### PR TITLE
LDAP module support

### PR DESCRIPTION
**Pull request by raphapr**

### CODE REVIEW

File: templates/rundeck-config.groovy.j2 [hunk 1]
```diff
@@ -41,3 +41,5 @@ rundeck.storage.converter."1".config.password = "{{jasypt_encryption_secret}}"
 rundeck.clusterMode.enabled = "{{ rundeck_clusterMode_enabled }}"
 
 rundeck.execution.logs.fileStoragePlugin = "org.rundeck.amazon-s3"
+
+rundeck.executionMode = "passive"
```
Comment (mtulio) [inline templates/rundeck-config.groovy.j2]: Should not set default as 'passive' mode.

File: templates/profile.j2 [hunk 2]
```diff
@@ -25,12 +25,10 @@ if [ ! -z $JAVA_HOME ]; then
     JAVA_CMD=$JAVA_HOME/bin/java
 fi
 
-
-
 export CLI_CP=$(find /var/lib/rundeck/cli -name \*.jar -printf %p:)
 export BOOTSTRAP_CP=$(find /var/lib/rundeck/bootstrap -name \*.jar -printf %p:)
 export RDECK_JVM="-Djava.security.auth.login.config=/etc/rundeck/jaas-loginmodule.conf \
-       -Dloginmodule.name=RDpropertyfilelogin \
+       -Dloginmodule.name={{ 'ldap' if rd_login_module == 'ldap' else 'RDpropertyfilelogin' }} \
```
Comment (mtulio) [inline templates/profile.j2]: This conditional check could be improved. http://jinja.pocoo.org/docs/2.9/templates/#if

**Review by mtulio — CHANGES_REQUESTED** [review]

File: templates/profile.j2 (hunk 2 above)
Comment (raphapr) [reply templates/profile.j2]: done

**Review by mtulio — APPROVED** [review]

**Comment by mtulio** [comment]
You could sqash and merge this PR. :)

**Pull request merged**